src/utils/tokens.ts

```diff
@@ -69,7 +69,7 @@ export async function getPoolTokensV3(
     try {
         const chainId = await client.getChainId();
         const vaultV3 = getContract({
-            abi: vaultV3Abi,
+            abi: vaultExtensionV3Abi,
```

Comment:
I guess `getPoolTokens` isn't part the Vault ABI anymore?

On scaffold balancer, I've been using the VaultExtension ABI instead of the Vault ABI everywhere. I tried pasting the VaultExtension ABI as the `vaultV3.ts` and the tests all seem to run fine. Would that be prefferrable instead of adding the VaultExtention ABI to the SDK?

Reply:
I can't remember if @brunoguerios did something to combine the abis?
If not I think its maybe best to keep it explicit to match the SCs so using VaultExtension.

Reply:
Yeah, I had combined the abis into a single one, but looking again at it, I think it's better if we keep them separated as you did 👍